src/components/shared/modals/BannyAttachModal.tsx

```diff
@@ -0,0 +1,77 @@
+import { t } from '@lingui/macro'
+import { Col, Modal, Row, Space } from 'antd'
+import { ThemeContext } from 'contexts/themeContext'
+import { useContext, useState } from 'react'
+
+type AttachableBanny = {
+  url: string
+  alt: string
+}
+
+const ATTACHABLE_BANNYS: AttachableBanny[] = [
+  { url: '/assets/banny_eth.png', alt: 'Blockchain Banny' },
+  { url: '/assets/banny_love.png', alt: 'Banny in love' },
+  { url: '/assets/banny_party.png', alt: 'Party banny' },
+  { url: '/assets/banny_popcorn.png', alt: 'Popcorn banny' },
+  { url: '/assets/quint.gif', alt: 'Quint' },
+  { url: '/assets/stoned_banny.png', alt: 'Stoned banny' },
+]
+
+export default function BannyAttachModal({
+  visible,
+  onBannySelected,
+  onClose,
+}: {
+  visible: boolean
+  onBannySelected: (url: string) => void
+  onClose: VoidFunction
+}) {
+  const {
+    theme: { colors },
+  } = useContext(ThemeContext)
+  const [selectedBannyIndex, setSelectedBannyIndex] = useState<number>()
+  const renderAttachableBanny = (banny: AttachableBanny, index: number) => {
+    return (
+      <Col md={8}>
+        <div
+          style={{
+            backgroundColor:
+              selectedBannyIndex === index ? colors.background.l2 : 'unset',
+            display: 'flex',
+            justifyContent: 'center',
+            alignItems: 'middle',
+            padding: '20px 0',
+            cursor: 'pointer',
+          }}
+          onClick={() => setSelectedBannyIndex(index)}
```

Comment:
I think you might be planning on doing this but just in case: are you cool with one-click add sticker and close modal as discussed in Discord?

Reply:
Doneski